libs/payments/cart/src/lib/cart.service.ts

```diff
@@ -47,6 +48,14 @@ export class CartService {
       ? this.geodbManager.getTaxAddress(args.ip)
       : undefined;
 
+    const eligibility = await this.eligibilityService.checkEligibility(
+      args.interval,
+      args.offeringConfigId,
+      stripeCustomerId
+    );
+
+    const cartEligibilityStatus = handleEligibilityStatusMap[eligibility];
```

Comment:
A follow up ticket will be created to build out a handler function so that the correct CartState, CartEligibilityStatus and CartErrorReasonId will be set on the cart, depending on the EligibilityStatus.

Reply:
To be completed by [FXA-9537](https://mozilla-hub.atlassian.net/browse/FXA-9537)